x/evm/ante/preprocess.go

```diff
@@ -2,20 +2,27 @@ package ante
 
 import (
 	"errors"
+	"math"
 	"math/big"
 
 	"github.com/btcsuite/btcd/btcec"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	accountkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
 	"github.com/ethereum/go-ethereum/common"
 	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 	"github.com/ethereum/go-ethereum/params"
 	evmkeeper "github.com/sei-protocol/sei-chain/x/evm/keeper"
 	evmtypes "github.com/sei-protocol/sei-chain/x/evm/types"
+	"github.com/sei-protocol/sei-chain/x/evm/types/ethtx"
 )
 
+// Accounts need to have at least 1Sei to force association. Note that account won't be charged.
```

Comment:
is this something enforced at the cosmos layer or a design decision we made? b/c accounts can be registered on chain with as little as 1usei right?

Reply:
@philipsu522 this is a decision we make to avoid ddos, since such transaction is gasless